src/CliModels.ts

```diff
@@ -0,0 +1,40 @@
+
+// ToDo: Change name
```

Comment:
what name?

Reply:
@jotamusik ?

Reply:
Removed, it was probably an outdated comment 😅